### PR TITLE
Edited CSV Writer

### PR DESCRIPTION
**Pull request by swang1997**

### CODE REVIEW

File: out.csv [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+23,B,not-in-use
+24,B,not-in-use
+25,B,not-in-use
+26,B,not-in-use
+27,B,not-in-use
+28,B,not-in-use
+29,B,not-in-use
+30,B,not-in-use
+31,B,not-in-use
+2222,A,not-in-use
```
Comment (Tejasvas123) [inline out.csv]: I dont think we need this file. This file should be added to .gitignore as we dont need git to keep track of it I guess. What do you think about it?

File: src/main/java/duke/logic/commands/ExportLockerCommand.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import duke.exceptions.DukeException;
+import duke.models.LockerList;
+import duke.storage.FileHandling;
+import duke.storage.OpenCsv;
+import duke.ui.Ui;
+
+public class ExportLockerCommand extends Command {
+
+    public ExportLockerCommand(){
```
Comment (Tejasvas123) [inline src/main/java/duke/logic/commands/ExportLockerCommand.java]: Try removing this constructor. I think we dont need to add this constructor as Java will automatically understand that you are calling the default constructor if you are not doing constructor overloading.

File: src/main/java/duke/parser/ExportLockerCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package duke.parser;
+
+import duke.logic.commands.ExportLockerCommand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExportLockerCommandParser {
+    private List<String> splitInput = new ArrayList<>();
```
Comment (Tejasvas123) [inline src/main/java/duke/parser/ExportLockerCommandParser.java]: Remove as this redundant and never used

File: src/main/java/duke/storage/OpenCsv.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import duke.models.Locker;
+
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.io.IOException;
+import java.util.List;
+
+public class OpenCsv {
+    private static final String STRING_ARRAY_SAMPLE = "data.csv";
```
Comment (Tejasvas123) [inline src/main/java/duke/storage/OpenCsv.java]: I think we should name it something like ```private static final String CSV_OUTPUT_PATH```

File: src/main/java/duke/storage/OpenCsv.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                    CSVWriter.DEFAULT_ESCAPE_CHARACTER,
+                    CSVWriter.DEFAULT_LINE_END);
+
+            String[] header = {"Locker", "Zone", "Status"};
+            csvWriter.writeNext(header);
+
+            for (Locker l : lockerList) {
+
+                String[] details = new String[header.length];
+                details[0] = l.getSerialNumber().getSerialNumberForLocker();
```
Comment (Tejasvas123) [inline src/main/java/duke/storage/OpenCsv.java]: Constants like 0 and stuff should be declared as private static final.
So here you can do something like ```private static final FIRST_COLUMN = 0``` (at the top as a member variable)
and then use it here as ```details[FIRST_COLUMN] = ...```

File: src/main/java/duke/parser/ExportLockerCommandParser.java (hunk 3 above)
Comment (swang1997) [reply src/main/java/duke/parser/ExportLockerCommandParser.java]: resolved

File: src/main/java/duke/logic/commands/ExportLockerCommand.java (hunk 2 above)
Comment (swang1997) [reply src/main/java/duke/logic/commands/ExportLockerCommand.java]: resolved

File: src/main/java/duke/storage/OpenCsv.java (hunk 4 above)
Comment (swang1997) [reply src/main/java/duke/storage/OpenCsv.java]: resolved

File: src/main/java/duke/storage/OpenCsv.java (hunk 5 above)
Comment (swang1997) [reply src/main/java/duke/storage/OpenCsv.java]: resolved

File: out.csv (hunk 1 above)
Comment (swang1997) [reply out.csv]: resolved

**Pull request merged**